src/libs/ReportActionsUtils.ts

```diff
@@ -265,6 +265,14 @@ function getOneTransactionThreadReportID(reportID: string, reportActions: OnyxEn
     return iouRequestActions[0].childReportID ?? null;
 }
 
+function isOptimisticAction(reportAction: ReportAction) {
```

Comment:
I think this also needs to handle the case where reportAction is undefined since we get the reportAction from an array index (typescript doesn’t help here). The old code did handle that case.